packages/core/RNSentryAndroidTester/app/src/androidTest/java/io/sentry/rnsentryandroidtester/RNSentryJsonUtilsTest.kt

```diff
@@ -0,0 +1,102 @@
+package io.sentry.rnsentryandroidtester
+
+import android.content.Context
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import com.facebook.react.bridge.JavaOnlyMap
+import com.facebook.soloader.SoLoader
+import io.sentry.react.RNSentryJsonUtils
+import org.json.JSONArray
+import org.json.JSONObject
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertNotNull
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class RNSentryJsonUtilsTest {
+    @Before
+    fun setUp() {
+        val context: Context = InstrumentationRegistry.getInstrumentation().targetContext
+        SoLoader.init(context, false)
+    }
```

Comment:
This should not be needed. Let's remove it, to know 100% that this can run before the RN initializes.